### PR TITLE
Trainings and project page is created. 

### PR DESCRIPTION
**Pull request by busratok**
### Description

- Training and project page is created
- How it works component is made reusable
- Helpers are reorganised
-

### CODE REVIEW

**Comment by sabata06** [comment]
image

I think it would look better if the headings were aligned in the picture I uploaded and the text would go down, let's approve this pr to edit it and edit it again if you want

**Comment by sabata06** [comment]
WhatsApp Image 2024-01-28 at 12 41 34

File: src/app/forcompanies/findyourtalent/page.tsx [hunk 1]
```diff
@@ -59,7 +32,7 @@ const ForCompanies = () => {
         headerText={"What we offer"}
         startingAt={true}
       />
-
+      <HowItWorks howItWorksData={howItWorksData} />
```
Comment (sabata06) [inline src/app/forcompanies/findyourtalent/page.tsx:35]: i added a comment with pictures

Comment (ilkaytech) [inline src/app/forcompanies/findyourtalent/page.tsx:35]: I made the How it works component on the Find your talents page. A small correction would fix the How it works component on the tranings and project page.

**Comment by busratok** [comment]
> WhatsApp Image 2024-01-28 at 12 41 34

I solved the issue.

File: src/app/forcompanies/findyourtalent/page.tsx (hunk 1 above)
Comment (busratok) [reply src/app/forcompanies/findyourtalent/page.tsx:35]: > I made the How it works component on the Find your talents page. A small correction would fix the How it works component on the tranings and project page.

I converted that component into a reusable one and fixed it.

Comment (sabata06) [reply src/app/forcompanies/findyourtalent/page.tsx:35]: i think its good thanks

Comment (sabata06) [reply src/app/forcompanies/findyourtalent/page.tsx:35]: its seems good thanks

**Review by sabata06 — APPROVED** [review]

**Review by ilkaytech — COMMENTED** [review]
looks beautiful

**Pull request merged**